### DISCUSSION TITLE
Azure w CV

### DISCUSSION THREAD

**Discussion by slusarczyk41** [AzureCommunityPL/group#208] [Ogólna]
Hej

Chce sobie przerabiać use case'y z implementacją na Azure, mam ich jakąś listę, są tam wymagania instrukcja i generalnie wszystko co jest potrzebne do nauki.

Pytanie moje jest takie: jak dokumentować takie projekty? Szukałem opcji żeby eksportować ustawienia wszystkich resources na danym koncie i wrzucić na githuba ale nie ma takiej opcji. 

Wiadomym jest ze lepiej w CV będzie wyglądać wpis za którym stoi jakiś konkretny kod lub ogólnie dowód że rzeczywiście to przerobiłem. 

Jakieś rady?

**Comment by maciejgos** [comment]
Według mnie GitHub + może jakiś wpis na blog albo sam GitHub i dobre README 😉

**Reply by aryll** [reply]
Moim zdaniem to zależy od roli. @slusarczyk41 wydaje mi się, że projekty o których piszesz to bardziej rola Azure Developera.

A w takich przypadkach nie spotkałam się, żeby ktokolwiek patrzył na takie projekty na GitHub (może tylko ja mam takie wrażenie?).
Ja widzę to w ten sposób: w ofercie Azure zazwyczaj jest dodatkiem do czegoś (np. .NET Core) i jako wymaganie które przedstawia rekruter jest 'Azure' a nie konkretny komponent.
Później na części technicznej rekrutacji ktoś może zadać jakieś pytania o Azure w stylu 'z jakimi zasobami pracowałeś' ale raczej rzadko wchodzi się w szczegóły i częściej cała rekrutacja skupiona jest na tym pierwszym elemencie (np. .NET Core). 

Moim zdaniem wylistowanie konkretnych zasobów w CV jest wystarczające - techniczne pytania mogą zweryfikować poziom wiedzy z danego obszaru.

Ale jasne, jak coś robisz to zawsze można wrzucić to na GitHub :)

**Reply by kamil-mrzyglod** [reply]
Ja też sie nie spotkałem aby ktokolwiek na to patrzył, co nie znaczy, że ktoś inny nie napotyka się na coś takiego przy każdej rozmowie :)

Z drugiej strony - jakiej wartości szukasz w dokumentowaniu tego typu zadań? Nie czuję za bardzo co sensownego można pokazać mając takie repozytorium. Na przykładzie - co pokażesz by udowodnić, że rozumiesz messaging na Azure? Pewnie, możesz mieć repo gdzie wrzucisz sobie jakiegoś konsumenta, może jakiś sprytny sposób dystrybucji zdarzeń - problem jest taki, że tak samo dobrze mogłoby to byc zaimplementowane np. z użyciem Kafki czy NServiceBusa. Masz pokazać, że umiesz rozwiązać problem, jeśli go już raz rozwiązałeś z użyciem technologii X to nie potrzebujesz miesiąca aby zrozumieć, jak zrobić to w Azure.

Podobne zagadnienie jak z certami - mogą być fajną podkładką na pozycje juniorskie / regular, wyżej tracą znaczenie bo nie liczy się, czy potrafisz wybudzony w nocy wyrecytować opis danej usługi, tylko jak dobrą masz intuicję jeśli chodzi o wybór narzędzia do rozwiązania problemu w odpowiedniej skali.
